http://theflamingbanker.blogspot.com/2010/10/death-of-thousand-tickets.html
http://bugs.gentoo.org/show_bug.cgi?id=342779

#
#
# patch "libpurple/protocols/oscar/family_chat.c"
#  from [9b3ad3b5c68ab5d3f33b0b856fdeaf898a5ed9a9]
#    to [397a3d8572c9863216cc7b37d26858f34a1583ab]
#
============================================================
--- libpurple/protocols/oscar/family_chat.c	9b3ad3b5c68ab5d3f33b0b856fdeaf898a5ed9a9
+++ libpurple/protocols/oscar/family_chat.c	397a3d8572c9863216cc7b37d26858f34a1583ab
@@ -88,6 +88,8 @@ infoupdate(OscarData *od, FlapConnection
 		return 1;
 	}
 
+	byte_stream_get16(bs); // skip the TLV count
+
 	/*
 	 * Everything else are TLVs.
 	 */
